update-publication/src/test/java/no/unit/nva/publication/update/UpdatePublicationHandlerTest.java

```diff
@@ -28,100 +27,88 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import org.mockito.stubbing.Answer;
 import org.zalando.problem.Problem;
 
-public class UpdatePublicationHandlerTest {
+public class UpdatePublicationHandlerTest extends ResourcesDynamoDbLocalTest {
 
     public static final String IDENTIFIER = "identifier";
     public static final JavaType PARAMETERIZED_GATEWAY_RESPONSE_PUBLICATION_RESPONSE_TYPE =
         objectMapper.getTypeFactory().constructParametricType(GatewayResponse.class, PublicationResponse.class);
     public static final JavaType PARAMETERIZED_GATEWAY_RESPONSE_PROBLEM_TYPE =
         objectMapper.getTypeFactory().constructParametricType(GatewayResponse.class, Problem.class);
-    public static final String OWNER = "owner";
-    public static final URI ANY_URI = URI.create("http://example.org/publisher/1");
+
     public static final String RESOURCE_NOT_FOUND_ERROR_TEMPLATE = "Resource not found: %s";
     public static final String SOME_MESSAGE = "SomeMessage";
-    
+
     private ResourceService publicationService;
     private Context context;
 
     private ByteArrayOutputStream output;
     private UpdatePublicationHandler updatePublicationHandler;
-    
+
     private Publication publication;
-    
+    private Environment environment;
+
     /**
      * Set up environment.
      */
     @BeforeEach
     public void setUp() {
-        Environment environment = mock(Environment.class);
+        super.init();
+
+        environment = mock(Environment.class);
         when(environment.readEnv(ALLOWED_ORIGIN_ENV)).thenReturn("*");
 
-        publicationService = mock(ResourceService.class);
+        publicationService = new ResourceService(client, Clock.systemDefaultZone());
         context = mock(Context.class);
 
         output = new ByteArrayOutputStream();
         updatePublicationHandler =
             new UpdatePublicationHandler(publicationService, environment);
         publication = createPublication();
     }
-    
-    @Test
-    @DisplayName("handler returns OK Response on ReportBasic input")
-    public void handlerReturnsOkResponseOnReportBasicInput() throws IOException, ApiGatewayException {
-        Path reportBasic = Path.of("ReportBasicTest.json");
-        Publication modifiedPublication = objectMapper
-                                              .readValue(IoUtils.stringFromResources(reportBasic), Publication.class);
-        serviceSucceedsAndReturnsModifiedPublication(modifiedPublication);
-        InputStream inputStream = generateInputStreamWithValidBodyAndHeadersAndPathParameters(
-            modifiedPublication.getIdentifier()).build();
-        updatePublicationHandler.handleRequest(inputStream, output, context);
-        GatewayResponse<PublicationResponse> gatewayResponse = toGatewayResponse();
-        assertEquals(SC_OK, gatewayResponse.getStatusCode());
-    }
-    
+
     @Test
-    @DisplayName("handler Returns OK Response On Valid Input")
```

Comment:
This test has changed to handler returns OK when Input is valid and user is the resource owner